lib/web_console/locale.rb

```diff
@@ -0,0 +1,2 @@
+require 'active_support/core_ext/string/access'
```

Comment:
Hey, thanks for doing this!

You can actually put it inside an initializer in the Railtie.